### PR TITLE
Handle invalid utf8

### PR DESCRIPTION
**Pull request by BrainMaestro**
Handles an edge case in #7

### CODE REVIEW

File: tests/snapshots/output/plain.snapshot.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ fn main() {
         "The perimeter of the rectangle is {} pixels.",
         perimeter(&rect1)
     );
+    println!(r#"This line contains invalid utf8:  "�����"#;
```
Comment (sharkdp) [inline tests/snapshots/output/plain.snapshot.txt]: Nice

**Comment by sharkdp** [comment]
Less code and more features - fantastic! :+1: - thank you very much.
